data-generator/generate_data_test.py

```diff
@@ -12,14 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import pytest
 import datetime
-
 from urllib.request import Request
 
-from util_compare_dicts import compare_dicts
-
 import generate_data
+import pytest
+from util_compare_dicts import compare_dicts
```

Comment:
Looks like `isort` wasn't able to categorize here.

`generate_data` and `util_compare_dicts` are local modules.

Reply:
Ahh yeah, had to specify a [`src` option](https://docs.astral.sh/ruff/settings/#src):

[14ecb94](https://github.com/mozilla-services/fourkeys/pull/77/commits/14ecb94a68cce1513d58fc05f869928a813b3826), [e09ddf3](https://github.com/mozilla-services/fourkeys/pull/77/commits/e09ddf3016fb87ecb9edbd3ec7280f956f0a1591)